test-data/unit/lib-stub/typing.py

```diff
@@ -61,6 +62,9 @@ class Awaitable(Generic[T]):
     @abstractmethod
     def __await__(self) -> Generator[Any, Any, T]: pass
 
+class AwaitableGenerator(Generator[T, U, V], Awaitable[V], Generic[T, U, V, S]):
```

Comment:
Because this stub (despite its extension) is actually a stub. It matches the real `typing.pyi`, not any real `typing.py`.